client/app/iplb/modal/offer/iplb-offer-change.html

```diff
@@ -1,18 +1,17 @@
-<cui-modal>
-    <form id="IplbOfferChangeForm">
-        <cui-modal-header data-on-dismiss="IpLoadBalancerOfferChangeCtrl.cancel()"></cui-modal-header>
-        <cui-modal-body data-title=":: 'iplb_modal_offer_change_title' | translate">
-            <oui-spinner data-ng-if="IpLoadBalancerOfferChangeCtrl.isModalLoading()"></oui-spinner>
-            <div data-ng-if="!IpLoadBalancerOfferChangeCtrl.isModalLoading()">
-                Old offer => {{ IpLoadBalancerOfferChangeCtrl.configuration.data.offer }}
-                <cui-modal-text data-text="'EDIT SOME OFFER HERE'"></cui-modal-text>
-            </div>
-        </cui-modal-body>
-        <cui-modal-footer>
-            <cui-cancel-button data-text="'common_cancel' | translate" data-on-click="IpLoadBalancerOfferChangeCtrl.cancel()"></cui-cancel-button>
-            <cui-submit-button data-text="'common_edit' | translate"
-                data-on-click="IpLoadBalancerOfferChangeCtrl.confirm()"
-                data-disabled="IplbOfferChangeForm.$invalid || IpLoadBalancerOfferChangeCtrl.isModalLoading()"></cui-submit-button>
-        </cui-modal-footer>
-    </form>
-</cui-modal>
+<form id="IplbOfferChangeForm"
+      name="IplbOfferChangeForm"
+      data-ng-submit="IplbOfferChangeForm.$valid && IpLoadBalancerOfferChangeCtrl.confirm()"
+      novalidate>
+    <oui-modal
+        data-title="{{ ::'iplb_modal_offer_change_title' | translate }}"
+        data-primary-label="{{ ::'common_edit' | translate }}"
+        data-secondary-action="IpLoadBalancerOfferChangeCtrl.cancel()"
+        data-secondary-label="{{ ::'common_cancel' | translate }}"
+        data-on-dismiss="IpLoadBalancerOfferChangeCtrl.cancel()"
+        data-loading="IpLoadBalancerOfferChangeCtrl.isModalLoading()">
+        <p class="oui-modal__text">
+            Old offer => {{ IpLoadBalancerOfferChangeCtrl.configuration.data.offer }}
+        </p>
+        <p class="oui-modal__text" data-text="'EDIT SOME OFFER HERE'"></p>
```

Comment:
I don't understand this line too

Reply:
@jleveugle The offer change part is not implemented yet. So they have put some dummy text which i have retained as it is and changed only the modal part

Reply:
IMO if it's not implemented, it should not be here.

Reply:
@cbourgois and @jleveugle  have removed the modal and added a todo for its implementation